### PR TITLE
Holowarrant Software Update

### PR DESCRIPTION
**Pull request by KingOfThePing**
Changes the holowarrant's displayed warrant to be in accordance of all other paperwork. Meaning some wording was changed to scrub NanoTrasen and the Aurora from the warrant itself.

### IC changelog
sccOS (formerly ntOS) has pushed an update to the ISD's holowarrants. They now properly display all information, including the branch of operation and the current vessel properly.

### CODE REVIEW

**Review by SleepyGemmy — APPROVED** [review]
code looks fine.

**Comment by KingOfThePing** [comment]
!review

File: html/changelogs/KingOfThePing - holowarrant_software_update.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ChangeMe
```
Comment (SleepyGemmy) [inline html/changelogs/KingOfThePing - holowarrant_software_update.yml:30]:
```suggestion
author: KingOfThePing
```

Comment (SleepyGemmy) [reply html/changelogs/KingOfThePing - holowarrant_software_update.yml:30]: or equivalent.

**Comment by KingOfThePing** [comment]
I forgor, but fixed, ty ty

**Review by mikomyazaki — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**